packages/SwingSet/src/controller.js

```diff
@@ -135,13 +136,16 @@ function makeEvaluate(e) {
 
 function makeSESEvaluator() {
   const evaluateOptions = makeDefaultEvaluateOptions();
-  const { transforms, ...otherOptions } = evaluateOptions;
+  const { transforms, shims = [], ...otherOptions } = evaluateOptions;
   const s = SES.makeSESRootRealm({
     ...otherOptions,
     transforms,
     consoleMode: 'allow',
     errorStackMode: 'allow',
+    shims: [SES1TameMeteringShim, ...shims],
+    configurableGlobals: true,
   });
+  const replaceGlobalMeter = SES1ReplaceGlobalMeter(s);
```

Comment:
From which Realm is the `replaceGlobalMeter` object?

We use `makeRequire` to get the `r`/`require` object from the kernel's Realm, because it's too hard to do it correctly from the outside. While we're still using SES1, we should make sure `SES1ReplaceGlobalMeter` returns an object from the new realm.

Reply:
I asked offline, he said kernel-realm.